pandas/core/frame.py

```diff
@@ -7416,23 +7419,22 @@ def quantile(self, q=0.5, axis=0, numeric_only=True,
 
         Returns
         -------
-        quantiles : Series or DataFrame
-
-            - If ``q`` is an array, a DataFrame will be returned where the
-              index is ``q``, the columns are the columns of self, and the
+        scalar, Series or DataFrame
+            - If `q` is an array, a DataFrame will be returned where the
+              index is `q`, the columns are the columns of self, and the
               values are the quantiles.
-            - If ``q`` is a float, a Series will be returned where the
+            - If `q` is a float, a Series will be returned where the
               index is the columns of self and the values are the quantiles.
 
         Examples
         --------
-
-        >>> df = pd.DataFrame(np.array([[1, 1], [2, 10], [3, 100], [4, 100]]),
-                              columns=['a', 'b'])
+        >>> df = pd.DataFrame(np.array([[1, 1], [2, 10], \
+                                        [3, 100], [4, 100]]), \
+                                        columns=['a', 'b'])
```

Comment:
We usually try to keep the examples small, but I think the results are almost meaningless with just two samples. I'd have may be 6 or 8 in this case. Also, we try to use data that looks kind of real, and that the user know before hand (see this example: https://github.com/pandas-dev/pandas/blob/master/pandas/core/frame.py#L7580).

Also, can you use the validations mentioned in the issue. This example can't be run, and the backslashes are redundant and a style error is probably being shown.

Reply:
Alright, @datapythonista . I'll think of something!

Reply:
Alright, @datapythonista . I'll think of something!

Reply:
@datapythonista please review my PR. I believe these examples should suffice.